quick_tour/namespacing.ipynb

```diff
@@ -17,7 +17,8 @@
    "source": [
```

Comment:
<p>not necessary but would be good to not have empty output cells following this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>